.nvmrc

```diff
@@ -1 +1 @@
-15.*
+16
```

Comment:
16.*? 16.13.0?

Reply:
Isn’t `16.*` the same as `16`? Do we need to pin the project to a specific minor version?